contents/kotlin/kotlin.md

```diff
@@ -0,0 +1,94 @@
+# Introduction to Kotlin
+
+Author: [Shradheya Thakre](https://github.com/tshradheya)
+
+**Chapter contents**
+
+* [Overview](#overview)
+* [Why Kotlin?](#why-kotlin)
+    * [Concise](#concise)
+    * [Safe](#safe)
+    * [Interoperable](#interoperable)
+    * [Tool Friendly](#tool-friendly)
+* [Kotlin for Android Apps](#kotlin-for-android-apps)
+    * [Advantages of Shifting to Kotlin](#advantages-of-shifting-to-kotlin)
+    * [Drawbacks of Shifting to Kotlin](#drawbacks-of-shifting-to-kotlin)
+
+# Overview
+
+Kotlin is a general purpose, open source, statically typed programming language for the JVM and Android that combines object-oriented and functional programming features.
+
+# Why Kotlin?
+
+### Concise
+
+Highly reduces the amount of boilerplate code.
+- A Java object with getters, setters, equals(), hashCode(), toString() and copy() can be created in a single line in Kotlin
+``` kotlin
+data class Customer(val name: String, val email: String, val company: String)
+```
+- Assigning values based on ranges is much simpler than Java. Below are extracts of code with similar logic in Java and Kotlin:
+
+**Kotlin**
+``` kotlin
+val quartile: Int
+quartile = when (playPercentage) {
+    in 0..24 -> 0
+    in 25..49 -> 1
+    ...
+}
+```
+
+**Java**
+``` java
+int quartile;
+if(playPercentage >= 0 && playPercentage <= 24) {
+    quartile = 0;
+} else if(playPercentage >= 25 && playPercentage <= 49) {
+    quartile = 1;
+}
+...
+```
+
+### Safe
```

Comment:
Need to add a new line here to be consistent

Reply:
I have added a new line everywhere before `###` so it's consistent. 
Am I missing your point in the comment? :P